prime-router/settings/organizations.yml

```diff
@@ -1157,7 +1157,7 @@
         credentialName: DEFAULT-SFTP
       timing:
         operation: MERGE
-        numberPerDay: 1440
+        numberPerDay: 360   # Useful when you need a bit more of: This thing all things devours: Birds, beasts, trees, flowers
```

Comment:
Alabama is now a nice test state to generate fake data for when you need tests that don't run every minute.